### PR TITLE
Add info on ws nodes being public IP address

### PR DESCRIPTION
**Pull request by TylerJewell**
Solves this issue.
https://github.com/codenvy/codenvy/issues/1779#event-962979360

### CODE REVIEW

File: src/main/_docs/admin-guide/admin-managing.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Overlay networks require a distributed key-value store. We embed ZooKeeper, a ke
 The default network in Docker is a "bridge" network. If your system will only require a single physical node, you can scale it with larger amounts of RAM using bridge networking (its default configuration).
 
 ## Scaling With Overlay Network (Linux Only)
+Each workspace node that you add will need to have Docker installed. It should be the same version of Docker that is installed on the master node running Codenvy. Additionally, the IP address of each workspace node must be publicly reachable by browsers. While each workspace is going to be clustered internally with Swarm, the nature of Swarm requires that each IP address is externally accessible.
```
Comment (garagatyi) [inline src/main/_docs/admin-guide/admin-managing.md]: Public IPs needed for users of WSs. not for swarm

File: src/main/_docs/admin-guide/admin-installation.md [hunk 2]
```diff
@@ -292,8 +292,11 @@ Install the most recent version of the Docker Engine for your platform using the
...
 
-## Ports and Architecture
-Codenvy's runtime launches a group of Docker containers in a compose relationship. The master node is where Codenvy is installed and running. In a [scalability mode]({{base}}/docs/admin-guide/managing/index.html), you can add additional physical "workspace" nodes to increase system capacity.
+## IP Addresses
+The hostname or IP address that you give to the Codenvy master node (and any optional workspace nodes) must be externally reachable by each browser. In scalability mode, you can create a cluster of workspace nodes by connecting different Docker daemons together. Even though the cluster is an internal object, each workspace node must be listening on a publicly reachable IP address or hostname.
+
+## Required Ports
+Codenvy's runtime launches a group of Docker containers in a compose relationship. The master node is where Codenvy is installed and running. In a [scalability mode]({{base}}/docs/admin-guide/managing/index.html), you can add additional physical "workspace" nodes to increase system capacity.
```
Comment (garagatyi) [inline src/main/_docs/admin-guide/admin-installation.md]: Can you elaborate on what is connection between this sentence and required ports?

File: src/main/_docs/admin-guide/admin-managing.md (hunk 1 above)
Comment (TylerJewell) [inline src/main/_docs/admin-guide/admin-managing.md]: Yes - I understand but I think it was a bit easier for admins to understand writing it this way.

File: src/main/_docs/admin-guide/admin-installation.md (hunk 2 above)
Comment (TylerJewell) [inline src/main/_docs/admin-guide/admin-installation.md]: In order to explain why the following sections are structured a certain way this sentence is there to provide the background necessary so the reader understands what is explained next.

**Comment by TylerJewell** [comment]
Thanks Alex.

**Pull request merged**